peerInvalid issue

Hi, when trying to install this module I get this issue:

```
npm ERR! peerinvalid The package typed-css-modules@0.1.10 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer typed-css-modules-loader@0.0.1 wants typed-css-modules@latest
```

I definitely have the latest version of `typed-css-modules` installed. Any clue what's happening?

--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
     "css-modules"
   ],
   "peerDependencies": {
-    "typed-css-modules": "latest"
+    "typed-css-modules": "^0.1.10"
   },
   "author": "Oleg Stepura",
   "license": "MIT",

